### DISCUSSION TITLE
how to config 3rd party libraries, such as sass_binary_site, phantomjs_cdnurl, electron_mirror, puppeteer, chromedriver_cdnurl

### DISCUSSION THREAD

**Discussion by hegphegp** [verdaccio/verdaccio#2294] [Q&A] [answered]
* how to config 3rd party libraries, such as sass_binary_site, phantomjs_cdnurl, electron_mirror, puppeteer, chromedriver_cdnurl

**Comment by juanpicado** [comment, marked as answer]
not possible, those binaries are downloaded outside of the context of a registry.
